src/Dotnet.Script.Core/ScriptPublisher.cs

```diff
@@ -85,7 +85,7 @@ public ScriptPublisher(LogFactory logFactory, ScriptEmitter scriptEmitter)
 
             var commandRunner = new CommandRunner(logFactory);
             // todo: may want to add ability to return dotnet.exe errors
-            var exitcode = commandRunner.Execute("dotnet", $"publish \"{tempProjectPath}\" -c Release -r {runtimeIdentifier} -o {context.WorkingDirectory}");
+            var exitcode = commandRunner.Execute("dotnet", $"publish \"{tempProjectPath}\" -c Release -r {runtimeIdentifier} -o \"{context.WorkingDirectory}\"");
```

Comment:
I think we should do the same around the nuget.config path? https://github.com/filipw/dotnet-script/blob/master/src/Dotnet.Script.DependencyModel/Context/DotnetRestorer.cs#L30

Reply:
Good catch. Probably also around the `packageSourcesArgument`